model/config.go

```diff
@@ -752,6 +753,19 @@ func (o *Config) SetDefaults() {
 		*o.EmailSettings.EmailBatchingInterval = EMAIL_BATCHING_INTERVAL
 	}
 
+	if o.EmailSettings.EnableSMTPAuth == nil {
+		o.EmailSettings.EnableSMTPAuth = new(bool)
+		if o.EmailSettings.ConnectionSecurity == CONN_SECURITY_NONE {
+			*o.EmailSettings.EnableSMTPAuth = false
+		} else {
+			*o.EmailSettings.EnableSMTPAuth = true
+		}
+	}
+
+	if o.EmailSettings.ConnectionSecurity == CONN_SECURITY_PLAIN {
```

Comment:
@hmhealey It definitely goes there. If you override the incorrect `ConnectionSecurity` setting from `None` to `Plain` then when it tries to set the `EnableSMTPAuth` it will set it incorrectly. `EnableSMTPAuth` must be set based on the user config before the config for `ConnectionSecurity` gets updated.